config/config.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"os"
+)
+
+type Config struct {
+	DiscordBotToken  string
+	DiscordChannelID string
+	DiscordMessageID string
+}
+
+func LoadConfig() *Config {
```

Comment:
Any reason for returns it as a pointer instead of value?

Reply:
nothing, just my usual convention, usually prefer pointers for memory efficiency